README.md

```diff
@@ -14,15 +15,17 @@ More user relevant information about the firmware are on the wiki page at: https
 For questions write a mail to <berlin@berlin.freifunk.net> or come to our weekly meetings.
 If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
 
+
 ## Development
 
 ### Info
 
-For the Berlin Freifunk firmware we use vanilla OpenWrt with additional patches
-and packages. The Makefile automates firmware
-creation and apply patches / integrates custom freifunk packages. All custom
-patches are located in *patches/* and all additional packages can be found at
-http://github.com/freifunk-berlin/packages_berlin.
+How does this stuff work? 
+
+The idea is to download OpenWRT via git, patch it with all patches in the `patches/` folder and configure it according to the configuration snippets in `configs`. 
+Then use the OpenWRT buildsystem to build all packages and the Imagebuilder and use the latter to assemble the final image with lists of packages in packages for the router.
+All additional packages can be found at http://github.com/freifunk-berlin/packages_berlin.
```

Comment:
I'm not sure, if there is some rule about line-breaks in the source, as it was in the above code before your change.

Reply:
Im not sure either. When I opened that in my editor, it looked fairly strange, so I changed it. For the parser at github that shouldn't matter anyway. But maybe we should consider, that this README.md might be opened in an editor on a terminal.

Do you think it would be a good idea, to assume a 80-char wide terminal?

Reply:
80 chars seem fine.